apps/skde/_posts/innhold/informasjonskapsler.md

```diff
@@ -0,0 +1,26 @@
+---
+filename: informasjonskapsler
+title: Informasjonskapsler
+lang: nb
+---
+
+Dette nettstedet benytter informasjonskapsler for å samle inn statistikk. Vi lagrer aldri informasjon som kan identifisere deg.
+
+Informasjonskapsler (cookies) er informasjonselementer som plasseres på datamaskinen din når du laster ned en nettside.
+
+Alle som besøker et nettsted skal ha informasjon om at nettstedet benytter informasjonskapsler. Du må godkjenne at informasjonskapsler blir benyttet. Godkjenningen gjøres gjennom innstillingene i din nettleser.
+
+Du kan hindre at slike informasjonskapsler lagres ved å endre på nettleserens innstillinger. Da vil imidlertid sidene ikke fungere optimalt. Du kan også slette informasjonskapsler i nettleseren eller ved filbehandling. [Slik administrerer du informasjonskapsler](https://nettvett.no/slik-administrer-du-informasjonskapsler/).
```

Comment:
Kan vi slette `Da vil imidlertid sidene ikke fungere optimalt.`? Stemmer kanskje ikke helt:)

Reply:
Ja, jeg så faktisk den der og lot den være for sikkerhetsskyld :p Men kan godt fjerne den, for jeg kunne ikke se noen andre cookies enn Matomo sine.

Reply:
Testen som feiler er for Cypress og "Velg register"-knappen, som jeg tror er urelatert til denne saken og tilfeldigvis krasjer av andre grunner. Kan man kjøre den på nytt og se om det går igjennom?

Reply:
Jeg kjører den på nytt for å se om den går igjennom.

Reply:
@arnfinn Jeg har fikset og sjekket inn:

"Da vil sidene imidlertid ..." er fjernet.
Helseatlas-filen er fjernet.
Rettet til SKDE